### PR TITLE
[ACS-5761] Demo Shell Cleanup (part 2)

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

- remove unused assets (Demo Shell)
- eliminate dead code (Demo Shell)
- remove unused features from Demo Shell
- fix https://alfresco.atlassian.net/browse/ADF-5478 with upload error tooltips
- Remove `About` page
- Remove `Custom Sources` page
- Greatly cleanup the APS component pages
- cleanup e2e tests for demo shell
  - remove the tests that already present in ACA, or replicating ACA to test the behaviour and not the component
  - remove e2e that already have unit tests

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://alfresco.atlassian.net/browse/ACS-5761

### CODE REVIEW

File: e2e/process-services/widgets/date-widget.e2e.ts [hunk 1]
```diff
@@ -39,12 +39,12 @@ describe('Date widget', () => {
     const loginPage = new LoginPage();
     const taskPage = new TasksPage();
     const widget = new Widget();
-    const navigationBarPage = new NavigationBarPage();
 
     const dateWidget = widget.dateWidget();
-    let appModel;
-    let processUserModel;
-    let deployedAppId; let process;
+    let appModel: any;
```
Comment (MichalKinas) [inline e2e/process-services/widgets/date-widget.e2e.ts:44]: Same here for type

File: e2e/process-services/widgets/date-widget.e2e.ts [hunk 2]
```diff
@@ -39,12 +39,12 @@ describe('Date widget', () => {
...
+    let processUserModel: UserModel;
+    let deployedAppId: number;
+    let process: any;
```
Comment (MichalKinas) [inline e2e/process-services/widgets/date-widget.e2e.ts:47]: Same here for type

File: e2e/process-services/tasks/start-task-custom-app.e2e.ts [hunk 3]
```diff
@@ -32,20 +31,18 @@ describe('Start Task - Custom App', () => {
...
-    const formFieldValue = 'First value ';
+    let processUserModel: UserModel;
+    let assigneeUserModel: UserModel;
+
     const taskPage = new TasksPage();
-    const firstComment = 'comm1'; const firstChecklist = 'checklist1';
     const tasks = ['Modifying task', 'Information box', 'No form', 'Not Created', 'Refreshing form', 'Assignee task', 'Attach File', 'Spinner'];
-    const showHeaderTask = 'Show Header';
-    let appModel;
+    let appModel: any;
```
Comment (MichalKinas) [inline e2e/process-services/tasks/start-task-custom-app.e2e.ts:44]: Same here for type

File: e2e/process-services/widgets/dynamic-table-widget.e2e.ts [hunk 4]
```diff
@@ -40,8 +41,9 @@ describe('Dynamic Table widget ', () => {
     const applicationsService = new ApplicationsUtil(apiService);
     const processUtil = new ProcessUtil(apiService);
 
-    let processUserModel;
-    let deployedAppId; let process;
+    let processUserModel: UserModel;
+    let deployedAppId: number;
+    let process: any;
```
Comment (MichalKinas) [inline e2e/process-services/widgets/dynamic-table-widget.e2e.ts:46]: Same here for type

File: e2e/process-services/widgets/date-time-widget.e2e.ts [hunk 5]
```diff
@@ -35,16 +35,16 @@ describe('Date and time widget', () => {
...
     const apiService = createApiService();
     const usersActions = new UsersActions(apiService);
     const applicationsService = new ApplicationsUtil(apiService);
     const processUtil = new ProcessUtil(apiService);
 
-    let processUserModel;
-    let appModel;
-    let deployedAppId; let process;
+    let processUserModel: UserModel;
+    let appModel: any;
```
[the remark this answers is not in the document]
Comment (DenysVuika) [reply e2e/process-services/widgets/date-time-widget.e2e.ts:45]: I can add a few types, but the whole protractor needs to be deleted eventually

[the remark this answers is not in the document]
Comment (MichalKinas) [reply e2e/process-services/widgets/date-time-widget.e2e.ts:45]: Right, both approaches are fine for me

File: e2e/process-services/widgets/dynamic-table-widget.e2e.ts (hunk 4 above)
Comment (DenysVuika) [reply e2e/process-services/widgets/dynamic-table-widget.e2e.ts]: done for all widgets there

**Review by MichalKinas — APPROVED** [review]

**Pull request merged**